src/features/unified-share-modal/flowTypes.js

```diff
@@ -140,3 +140,9 @@ export type collaboratorType = {
 export type collaboratorsListType = {
     collaborators: Array<collaboratorType>,
 };
+
+export type tooltipComponentIdentifierType =
```

Comment:
just componentIdentifierType?

Reply:
nvm this makes more sense since it restricts the possible values for the tooltip prop